public/data/omkar009.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Omkar Ramishte",
+  "type": "personal",
+  "bio": "Currently Cs Grad, beginning with open source journey, learner, share about open source and threads and also Blog on Twitter. Make sure to check out. Happy to connect❤️ ",
+  "avatar": "https://github.com/omk-coder.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/omk-coder",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/omkar-ramishte-9ba784207/",
+      "icon": "LinkedIn"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ORamishte",
+      "icon": "twitter"
+    }
+  ],
+  "milestomes": [
+    {
+      "url": "https://www.eddiejaoude.io/omkar009"
```

Comment:
Please update your milestone section,  you're required to follow the **syntax** given in our readme. 
Something like:

```json
  "milestones": [
    {
      "title": "Started Freelancing",
      "date": "December 2021",
      "icon": "dollar",
      "color": "grey",
      "description": "Started freelancing",
      "url": "https://www.eddiejaoude.io/"
    }
  ]
```

Another thing, this is a broken URL, use another in place of this! Let us know if you meed any help with this!

Reply:
You can also remove milestone section, if you want! 🙃